### PR TITLE
Remove ACS population subclasses and run both state and county level from the same DataSource. This simplifies ingestion configuration

### PR DESCRIPTION
**Pull request by aaronsn**

### CODE REVIEW

File: python/datasources/acs_population.py [hunk 1]
```diff
@@ -129,39 +129,29 @@ def update_col_types(frame):
...
+class ACSPopulationIngester():
     """American Community Survey population data in the United States from the
        US Census."""
 
-    def __init__(self, county_level):
-        # TODO pass this in from message data.
-        self.base_acs_url = "https://api.census.gov/data/2019/acs/acs5"
+    def __init__(self, county_level, base_acs_url):
+        self.base_acs_url = base_acs_url
         """The base ACS url to use for API calls."""
```
Comment (jenniebrown) [inline python/datasources/acs_population.py:138]: I don't think it's common practice to write docstring-style comments underneath member variables. Could you move the comments to above their variables and use '#' style comments instead?

File: python/datasources/acs_population.py [hunk 2]
```diff
@@ -360,21 +348,32 @@ def get_sex_by_age_and_race(self, var_map, sex_by_age_frames):
         return result
 
 
-class ACSStatePopulation(ACSPopulationBase):
-    def __init__(self):
-        super().__init__(False)
+# TODO pass this in from message data.
+BASE_ACS_URL = "https://api.census.gov/data/2019/acs/acs5"
```
Comment (jenniebrown) [inline python/datasources/acs_population.py:352]: nit: could you move this to the top of the file?

**Review by jenniebrown — CHANGES_REQUESTED** [review]
Is there ever a situation where we might want to ingest one and not the other?

File: python/datasources/acs_population.py (hunk 1 above)
Comment (aaronsn) [reply python/datasources/acs_population.py:138]: Huh I thought I read this somewhere. Oh well, updated to # style

**Comment by aaronsn** [comment]
> Is there ever a situation where we might want to ingest one and not the other?

In theory, maybe, but I can't think of any reason it would be that important to do so. ACS releases state/county data together so we can do them on the same schedule. I realized this introduces additional annoyances with the airflow configuration and I don't think it buys us that much.

**Review by jenniebrown — APPROVED** [review]

**Pull request merged**